cypress/utils/plugins/dashboards-assistant/commands.js

```diff
@@ -152,3 +153,14 @@ Cypress.Commands.add('stopDummyServer', () => {
     }
   });
 });
+
+Cypress.Commands.add('sendMessage', (body) =>
```

Comment:
will this be too generic?  trying to think if there might be a time where we want to avoid a conflict but I believe assistant is the only thing with a `message` or `conversation`

Reply:
Will change to `sendAssistantMessage`.